app/public/index.html

```diff
@@ -1,30 +1,10 @@
 <!DOCTYPE html>
-<!-- Copyright 2022 NMIND
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    https://github.com/nmind/standards-checklist/blob/main/LICENSE
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License. -->
```

Comment:
I don't really care if we drop the license notice comment, but I think it's out of scope for this PR

Reply:
Copied it over in [02d7c65](https://github.com/nmind/standards-checklist/pull/30/commits/02d7c65a98cddacc5e9d1730ed0f136de168e5b8)